File: my-folder/0001-two-sum/solution.py
class Solution(object):
    def twoSum(self, nums, target):
        
        for i in range(len(nums)):
            number = target - nums[i]
            if number in nums:
                j=nums.index(number)    # arr.index() is function to find the index of num
                if j!=i:       # j and i are the indexes
                    return[i,j]

File: my-folder/0001-two-sum/solution.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        int n = nums.length;
        Map<Integer,Integer> mp = new HashMap<>();

        for(int i=0 ; i<n ; i++){
            mp.put(nums[i],i);
        }


        for(int i=0 ; i<n ; i++){
            int number = target-nums[i];
            if(mp.containsKey(number) && mp.get(number)!=i){
                return new int[] {i,mp.get(number)};
            }
        }
        return new int[]{};
    }
}
